cfgov/unprocessed/css/enhancements/forms.less

```diff
@@ -361,6 +361,14 @@ input {
     } );
 }
 
+/*
+ * Removes top margin from text input when it follows
+ * .a-label__heading, which applies a 10px bottom margin.
+ */
+.m-form-field .a-label__heading + .a-text-input {
+    margin-top: 0;
```

Comment:
@sebworks CF issue here: https://github.com/cfpb/capital-framework/issues/796

Reply:
Interesting. What do you think about consistently setting the spacing as margin top on the a-text-input (removing the 10px from a-label__heading) rather than adding/removing in different places?

Reply:
@jimmynotjim It looks like we're currently relying on the `a-label__heading` bottom margin when we use it with different form elements (like the Topic/Author multi-selects in the blog filters). I definitely think it would be simpler if we consistently set the margin either on the label or the form inputs, though.

Reply:
Agreed. Mind adding a TODO to try to consolidate the spacing in the future?

Reply:
Added in https://github.com/cfpb/cfgov-refresh/pull/4005/commits/5ea8b2f1215143ee19b9af3bc7553f8e4aec6b3f